.github/workflows/generateChangelog.yml

```diff
@@ -0,0 +1,35 @@
+name: Generate Changelog
+
+on:
+  workflow_run:
+    workflows: ['Create Release Branch']
+    types:
+      - completed
```

Comment:
Does this also indicate the workflow was successful ?

Reply:
@diyer **type:completed** means the workflow finished running, regardless of whether it is successful or not.  It is also used in e2e.yml to start an E2E tests job whenever VSIXs are available after _Nightly Build Develop_ or _Test, Build, and Release_.

Documentation here: https://docs.github.com/en/webhooks/webhook-events-and-payloads#workflow_run

NOTE: There are multiple ways for one workflow to call another - this is just one method, and a perfectly valid one that has proven to work for running E2E tests with VSIXs.  I can think of another method that would seem equally valid, using **jobs** at the bottom of the caller workflow.  Based on what I see in the Apex E2E Tests workflow, I believe that one can filter on success vs. failure of the _Create Release Branch_ workflow - although I did not test that, so this is just my speculation based on what I have been doing with the E2E test workflows in the past.  I'm sure @mingxuanzhangsfdx  has his reasoning for why he chose one method of calling workflows over another.

Reply:
Thank you @diyer @daphne-sfdc . I made changes to make sure the job create_branch succeeds and trigger the changelog generation job.